guake/paths.py.in

```diff
@@ -94,3 +94,5 @@ AUTOSTART_FOLDER = {{ AUTOSTART_FOLDER }}
 def try_to_compile_glib_schemas():
     log.info("Compiling schema: %s", SCHEMA_DIR)
     subprocess.check_call(["glib-compile-schemas", "--strict", SCHEMA_DIR])
+    with open(os.path.join(SCHEMA_DIR, "version"), "w", encoding="utf-8") as version:
```

Comment:
Prefer pathlib, and I don't think we need `encoding="utf-8"`, they won't contain unicode codepoint character.

Reply:
Including encoding in open() calls makes the linter happy. I'll change it to ascii.

Reply:
emit by flake8?

Reply:
I changed the explicit encoding to ascii already. Probably don't want to remove the check from our linter, it's an alright check